### ISSUE TITLE
Google Drive links not working

### ISSUE THREAD

**Issue by artur-ag** [miquelmarti/Okutama-Action#8]
Google made an [update to some Google Drive file links](https://workspaceupdates.googleblog.com/2021/06/drive-file-link-updates.html), and I think the links in this repo were affected. Could you update those links (training set, test set, trained models)?

From what I can tell, it's necessary to recreate the link from Google Drive. The ID will be the same, but the link will have an extra parameter `?resourcekey=xxxxxxxxxxxx`.

**Comment by miquelmarti** [comment]
Thanks for pointing this out @artur-ag.

@Mohammadamin-Barekatain should we maybe host the dataset elsewhere instead to ease the process of downloading it?
